### PR TITLE
Content tweaks

### PR DESCRIPTION
**Pull request by frankieroberto**
Content and design updates following review, mainly comprising:

* error message wording
* remove "cosmetic" from "cosmetic product" in headings
* changing "component" to "item"
* fixing minor layout issues

Review app: https://cosmetics-pr-1453-submit-web.london.cloudapps.digital

### CODE REVIEW

File: cosmetics-web/app/views/responsible_persons/notification_files/new.html.slim [hunk 1]
```diff
@@ -11,7 +11,7 @@
       span.error-summary-js
         - if @errors.blank?
           = render "components/govuk_error_summary", titleText: "There is a problem", errorList: [{ text: "Too many" + \
-              " files selected. Please select no more than #{NotificationFile.get_max_number_of_files} files",
+              " files selected. You can only select up to #{NotificationFile.get_max_number_of_files} files",
```
Comment (PabCab) [inline cosmetics-web/app/views/responsible_persons/notification_files/new.html.slim]: `errorList: [{ text: "Too many" + \
              " files selected. You can only select up to #{NotificationFile.get_max_number_of_files} files",`

Remove Too many" + \
              " files selected. 


`errorList: [{ text: "You can only select up to #{NotificationFile.get_max_number_of_files} files at the same time",`

File: cosmetics-web/app/views/component_build/add_component_name.html.slim [hunk 2]
```diff
@@ -18,5 +18,5 @@ p
               form: form,
               key: :name,
               label: { text: question, classes: "govuk-label--m" },
-              hint: { text: 'For example, ’Cream peroxide’' }
+              hint: { text: 'For example, ’cream peroxide’' }
```
[the remark this answers is not in the document]
Comment (frankieroberto) [reply cosmetics-web/app/views/component_build/add_component_name.html.slim]: Well spotted! My bad.

File: cosmetics-web/app/views/notification_build/single_or_multi_component.html.slim [hunk 3]
```diff
@@ -1,30 +1,38 @@
...
+      ul.govuk-list.govuk-list--bullet
+        li hair dye kits
+        li skin peeling kits
+        li permanent styling products
+        li anti-ageing or revitalising kits
+      p.govuk-inset-text
+        | Where a set contains separate products that do not need to be
+          mixed together, or used one after the other in a particular order,
+          for the product to work, you need to
+        =< link_to "submit a notification for each product separately", \
```
Comment (PabCab) [inline cosmetics-web/app/views/notification_build/single_or_multi_component.html.slim]: `=< link_to "submit a notification for each product separately", \`

Change to:

 ` =< link_to "submit a separate notification for each product", \`

File: cosmetics-web/app/views/notification_build/single_or_multi_component.html.slim [hunk 4]
```diff
@@ -1,30 +1,38 @@
...
+        | Examples of multi-item kits include:
+      ul.govuk-list.govuk-list--bullet
+        li hair dye kits
+        li skin peeling kits
+        li permanent styling products
+        li anti-ageing or revitalising kits
+      p.govuk-inset-text
+        | Where a set contains separate products that do not need to be
+          mixed together, or used one after the other in a particular order,
+          for the product to work, you need to
```
Comment (PabCab) [inline cosmetics-web/app/views/notification_build/single_or_multi_component.html.slim]: Change

`for the product to work, you need to`

to

`for the product to work, you'll need to`

File: cosmetics-web/app/models/notification.rb [hunk 5]
```diff
@@ -179,7 +179,13 @@ def all_required_attributes_must_be_set
 
     changed.each { |attribute|
       if mandatory_attributes.include?(attribute) && self[attribute].blank?
-        errors.add attribute, "must not be blank"
+
+        # TODO: refactor this by moving validation to the model.
+        if attribute == "product_name"
+          errors.add attribute, "Enter the product name"
+        else
+          errors.add attribute, "must not be blank"
```
Comment (PabCab) [inline cosmetics-web/app/models/notification.rb]: Ideally we shouldn't have a coverall error message, but if we're going to have one....

Change  

`errors.add attribute, "must not be blank"`

to

 `errors.add attribute, "Must not be empty"`

File: cosmetics-web/app/controllers/concerns/manual_notification_concern.rb [hunk 6]
```diff
@@ -53,7 +53,19 @@ def yes_no_question(param, no_is_to_skip: true, before_skip: nil, before_render:
       before_render&.call
       render_wizard model
     else
-      model.errors.add param, "Choose an option"
+      if param == :is_hair_dye
```
Comment (slorek) [inline cosmetics-web/app/controllers/concerns/manual_notification_concern.rb]: Consider using a `case param` statement here, e.g.

```ruby
error_message = case param
  when :is_hair_dye
    "Select yes if the product contains a hair dye"
  ...
end

model.errors.add param, error_message

File: cosmetics-web/app/controllers/responsible_persons/add_notification_wizard_controller.rb [hunk 7]
```diff
@@ -16,7 +16,16 @@ def show
   def update
     answer = params[:answer]
     if answer != "yes" && answer != "no"
-      @error_text = "Please select an answer"
+
+      @error_text = if step == :do_you_have_files_from_eu_notification
```
Comment (slorek) [inline cosmetics-web/app/controllers/responsible_persons/add_notification_wizard_controller.rb]: A `case` statement would be useful here, too.

File: cosmetics-web/app/models/notification.rb [hunk 8]
```diff
@@ -185,7 +185,13 @@ def all_required_attributes_must_be_set
 
     changed.each { |attribute|
       if mandatory_attributes.include?(attribute) && self[attribute].blank?
-        errors.add attribute, "must not be blank"
+
+        # TODO: refactor this by moving validation to the model.
```
Comment (slorek) [inline cosmetics-web/app/models/notification.rb]: Which model?

File: cosmetics-web/app/controllers/concerns/manual_notification_concern.rb (hunk 6 above)
Comment (frankieroberto) [reply cosmetics-web/app/controllers/concerns/manual_notification_concern.rb]: Good point. This started out as a simple if statement, but then grew... 😀

File: cosmetics-web/app/models/notification.rb (hunk 8 above)
Comment (frankieroberto) [reply cosmetics-web/app/models/notification.rb]: I'll remove this comment. I was intending to refactor these odd private method (`all_required_attributes_must_be_set` and `mandatory_attributes`) to more conventional Rails validations. But that can wait for another day...

**Pull request merged**